frontend/components/Blocks/SectionBlock/ContentColumn.tsx

```diff
@@ -45,19 +45,27 @@ export default function ContentColumn({
 
   function getDefaultValue(content: InteractiveContent): string | number | string[] | undefined {
     const defaultValue = content.value.defaultValueOverride;
+    const targetValue = content.value.targetValue;
+
     switch (content.value.type) {
       case "single_select":
         if (defaultValue) {
           return content.value.options.find(
             option => option.option === defaultValue || option.label === defaultValue
           )?.id;
+        } else if (targetValue) {
+          return content.value.options.find(
+            option => option.option === targetValue || option.label === targetValue
+          )?.id;
```

Comment:
Mijn enige twijfel zit erin of dit de gewenste functionaliteit is, volgens mij gaat target value namelijk alleen maar over de waardes in andere secties. Met deze toevoeging wordt de target value ook als default gebruikt terwijl we een default value veld hebben hiervoor. @MariaBee3 zou je dit voor de zekerheid met Seth willen valideren?